src/GovUk.Education.ExploreEducationStatistics.Common/Services/TimePeriodLabelFormatter.cs

```diff
@@ -99,7 +99,7 @@ private string FormatYearInternal(string year)
                     case YearFormatOption.Default:
                         return year;
                     case YearFormatOption.AcademicOrFiscal:
-                        return $"{Parse(year)}/{(Parse(year) % 100) + 1}"; // Only want the last two digits
+                        return $"{Parse(year)}/{(Parse(year) + 1) % 100:D2}"; // Only want the last two digits
```

Comment:
I added Reporting Year recently - is that OK as 4 digit?

Reply:
Yes, I've just updated the debug test and checked the display of it